### PR TITLE
Dose

### PR DESCRIPTION
**Pull request by opotowsky**
1. Adding a function to Material that calculates dose per gram when given dose type (required, given as a string) and data source (optional) as input. 
2. Added Bq to Ci and Ci to Bq constants.

Feel free to suggest an alternative design.

### CODE REVIEW

**Comment by scopatz** [comment]
@opotowsky Is this something you would like to see in v0.5?

**Comment by opotowsky** [comment]
I got stuck on getting this to work then I had a month of no work so dropped the ball. I'm not sure how to implement this python version data type stuff.

**Comment by scopatz** [comment]
Oh. What are you getting hung up on exactly? We are going to try to get a pyne release out in the near future and it would be great if this made it in....

**Comment by opotowsky** [comment]
@scopatz I double checked to make sure I didn't have any merge conflicts but apparently I have them still. I don't know how to check this again because a pull tells me everything is up to date with develop

**Comment by scopatz** [comment]
Oh nope.  You don't have any conflicts anymore.  If you can merge in from upstream pyne develop without problems then you are good to go.

**Comment by scopatz** [comment]
I have PR'd a fix to you https://github.com/opotowsky/pyne/pull/5

**Comment by opotowsky** [comment]
Got it, checked, and merging. Thanks! I'm keeping an eye on the CI

**Comment by crbates** [comment]
Tests look good. All I see left is a minor docstring cleanup.

File: pyne/material.pyx [hunk 1]
```diff
@@ -23,6 +23,7 @@ from pyne.utils import QAWarning
 import os
 import sys
 if sys.version_info[0] >= 3:
+    #Python2 basestring is now Python3 string
```
Comment (opotowsky) [inline pyne/material.pyx]: @crbates is this what you are referring to?

File: src/material.h [hunk 2]
```diff
@@ -220,13 +220,23 @@ namespace pyne
...
+    comp_map decay_heat();
+    /// Caclulates the dose per gram using the composition of the the
+    /// material, the dose type desired, and the source for dose factors
+    ///   dose_type is one of:
+    ///     ext_air -- returns mrem/h per g per m^3
+    ///     ext_soil -- returns mrem/h per g per m^2
+    ///     ingest -- returns mrem per g
+    ///     inhale -- returns mrem per g
+    ///   source is: 
+    ///     {EPA=0, DOE=1, GENII=2}, default is EPA
```
Comment (opotowsky) [inline src/material.h]: @crbates or is this what you are referring to?

File: pyne/material.pyx [hunk 3]
```diff
@@ -618,6 +618,29 @@ cdef class _Material:
         return nucvec_proxy
 
 
+    def dose_per_g(self, dose_type, source=0):
+        """This provides the dose per gram using the comp of the the Material.
+        ext_air_dose returns mrem/h per g per m^3
+        ext_soil_dose returns mrem/h per g per m^2
+        ingest_dose returns mrem per g
+        inhale_dose returns mrem per g
```
Comment (crbates) [inline pyne/material.pyx]: sorry, I thought I added a line comment here earlier. I was thinking these lines here should either be in the `nucvec` returns section or that should be updated to make it clear what you are returning.

Comment (opotowsky) [inline pyne/material.pyx]: aha! on it

File: pyne/material.pyx [hunk 4]
```diff
@@ -622,6 +622,36 @@ cdef class _Material:
...
+            0 for EPA, 1 for DOE, 2 for GENII
+
+        Returns
+        -------
+        nucvec : dict
+            For a Material mat:
+            ext_air_dose returns mrem/h per g per m^3
+            ext_soil_dose returns mrem/h per g per m^2
+            ingest_dose returns mrem per g
+            inhale_dose returns mrem per g
```
Comment (opotowsky) [inline pyne/material.pyx]: @crbates Is this sufficient?

**Comment by crbates** [comment]
looks good, just waiting on CI.

**Comment by opotowsky** [comment]
Thanks so much for the help!

**Comment by crbates** [comment]
thanks @opotowsky!

**Pull request merged**